### PR TITLE
Major update of de.json

### PR DESCRIPTION
**Pull request by alixunderplatz**
- add new & missing zones
- add missing strings
- rephrase some German strings

I hope I didn't mess anything up! ^^

### CODE REVIEW

File: web/locales/de.json [hunk 1]
```diff
@@ -64,1078 +104,1138 @@
...
             "zoneName": "Dschibuti"
-          },
-          "DK-DK1": {
+        },
+        "DK": {
+            "zoneName": "Dänemark"
+        },
+        "DK-DK1": {
             "countryName": "Dänemark",
             "zoneName": "West"
```
Comment (jarek) [inline web/locales/de.json]: I think this could be updated for consistency with Norway, Brazil, etc? DK2 as well

Comment (alixunderplatz) [reply web/locales/de.json]: Thanks! Fixed it :)

File: web/locales/de.json [hunk 2]
```diff
@@ -6,19 +6,51 @@
...
+        "view1": {
+            "header": "Electricity Map",
+            "subtitle": "Visualisierung der klimarelevanten Emissionen der Stromerzeugung"
+        },
+        "view2": {
+            "header": "Betrachte in Echtzeit, wie viel CO2-Äquivalente bei der Stromerzeugung freigesetzt werden.",
+            "text": "Die Färbung der Zonen erfolgt anhand der spezifischen Emissionen klimawirksamer Gase (CO2-Intensität), die zur Deckung des Strombedarfs in einer Region freigesetzt werden. Eine grüne Färbung steht dabei für eine klimafreundlichere Erzeugung."
+        },
+        "view3": {
+            "header": "Erfahre, aus welchen Quellen deine Elektrizität stammt.",
```
Comment (jarek) [inline web/locales/de.json]: view2 and view3 headers have periods at the end, view1 and view4 don't

File: web/locales/de.json [hunk 3]
```diff
@@ -64,1078 +104,1138 @@
...
+        "SR": {
             "zoneName": "Suriname"
-          },
-          "SS": {
-            "zoneName": "Südsudan"
-          },
-          "ST": {
+        },
+        "SS": {
+        "zoneName": "Südsudan"
```
Comment (jarek) [inline web/locales/de.json]: indent looks different

**Comment by corradio** [comment]
@jarek what is the status here?

**Pull request merged**